documentation/book/proc-enabling-tracing-in-connect-mirror-maker-resources.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// assembly-setting-up-tracing-mirror-maker-connect.adoc
+
+[id='proc-enabling-tracing-in-connect-mirror-maker-resources-{context}']
+= Enabling tracing in the KafkaMirrorMaker, KafkaConnect, and KafkaConnectS2I custom resources
+
+Update the configuration of the `KafkaMirrorMaker`, `KafkaConnect`, and `KafkaConnectS2I` custom resources to specify and configure a Jaeger tracer service. When a tracing-enabled resource is updated in your Kubernetes cluster, two actions are triggered:
+
+* The Interceptors for the consumers and producers that are integrated into Mirror Maker or Kafka Connect are updated.
+
+* The tracing agent initializes a Jaeger tracer based on the tracing configuration in the resource.
+
+.Procedure
+
+Perform these steps for each `KafkaConnect`, `KafkaConnectS2I`, and `KafkaMirrorMaker` resource.
+
+. In the `spec.template` property, configure the Jaeger tracer service. Use the xref:ref-tracing-environment-variables-{context}[tracing environment variables] as template configuration properties. For example:
```

Comment:
Maybe I missed that but overall the documentation I don't see any paragraph/sentence saying that you have to install a Jaeger tracer service in the cluster so that it's a requirement and you can use it for configuring our Strimzi resources.

Reply:
Good point. So, we describe installing a Jaeger tracer service in "Initializing a Jaeger tracer for Kafka clients". I will add this as a prerequisite here.

Reply:
Or are we missing a whole procedure for deploying an instance of Jaeger to the Kafka cluster? @scholzj - please advise.

Reply:
I guess Paolo means the Jaeger backend which will collect the traces and whihc has the UI for analyzing them. I guess that should be somewhere in prerequisites and we can point to some docs. But I'm not sure we should document that. It makes as dependent on any of their changes which we would need to check etc.